core/src/main/java/org/mule/runtime/core/internal/event/DefaultEventBuilder.java

```diff
@@ -55,6 +55,7 @@
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import java.io.ObjectInputStream;
 import java.io.ObjectOutputStream;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.io.ObjectOutputStream'. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7752&issues=AWm_hBy7otIXnmsUvMwY&open=AWm_hBy7otIXnmsUvMwY)

[ISSUE_KEY]: AWm_hBy7otIXnmsUvMwY